src/pages/list/[id]/index.tsx

```diff
@@ -64,7 +65,11 @@ export default function SingleList() {
   }, [loadingListData, isLoading, setAdmins, listData, data]);
 
   return (
-    <PageWithBanner>
+    <PageLayout
+      title={listDetails?.name ?? ""}
+      description={listDetails?.description ?? ""}
+      image={listDetails?.image ?? ""}
+    >
```

Comment:
`<PageWithBanner>` should stay on individual routes

```suggestion
    >
    <PageWithBanner>
```

Reply:
done